### PR TITLE
WINDUP-3257: analyzeKnownLibraries

### PR DESCRIPTION
**Pull request by RichardHoch**
MTV 5.3.0

Resolves https://issues.redhat.com/browse/WINDUP-3257

Add information about the new advanced option, analyzeKnownLibraries, to the MTA documentation. 

Previews: 
1. Web console guide https://deploy-preview-529--windup-documentation.netlify.app/docs/web-console-guide/master/index.html#web-create-project_ocp-42 [right before step 16]. 

2. CLI guide: https://deploy-preview-529--windup-documentation.netlify.app/docs/cli-guide/master/index.html#cli-args_cli-guide [Third item in table]. 

3. Maven plugin guide: https://deploy-preview-529--windup-documentation.netlify.app/docs/maven-guide/master/index.html#maven-arguments_maven-guide [First item in table].

### CODE REVIEW

File: docs/topics/web-create-project.adoc [hunk 1]
```diff
@@ -65,14 +65,26 @@ Registering the server path ensures that the {ProductShortName} server always us
...
 * *Export CSV*: Exports the report data as a CSV file.
 * *Disable Tattletale*: Disables generation of a Tattletale report for each application.
-* *Class Not Found analysis*: Enables analysis of Java files that are not available on the class path. This option should not be used if some classes are unavailable for analysis.
+* *Class Not Found analysis*: Enables analysis of Java files that are not available on the class path.
++
+[NOTE]
+====
+This option should not be used if some classes are unavailable for analysis.
+====
+
```
Comment (RichardHoch) [inline docs/topics/web-create-project.adoc:74]: I broke out the second sentence as a note to be consistent with anayzeKnownLibraries; there's no change to the actual description of this option.

**Review by sshveta — APPROVED** [review]
Looks good :+1:

File: docs/topics/cli-args.adoc [hunk 2]
```diff
@@ -24,6 +24,13 @@ To run the {ProductShortName} command without prompting, for example when execut
 |Argument |Description
 |--additionalClassPath |A space-delimited list of additional JAR files or directories to add to the class path so that they are available for decompilation or other analysis.
 |--addonDir |Add the specified directory as a custom add-on repository.
+|--analyzeKnownLibraries | Analyze known software artifacts embedded within your application. By default {ProductShortName} only analyzes application code.
+
```
Comment (sbeskin-redhat) [inline docs/topics/cli-args.adoc:28]: @RichardHoch  
All arguments in this table used for choosing existing options are called flags ("Flag to..."). Why isn't this one a flag? "Flag to analyse known..."
Also, a comma after "By default".

File: docs/topics/maven-arguments.adoc [hunk 3]
```diff
@@ -12,6 +12,13 @@ The following is a detailed description of the available {ProductShortName} {Mav
 [cols="40%,60%a",options="header"]
 |====
 |Argument |Description
+|analyzeKnownLibraries | Analyze known software artifacts embedded within your application. By default {ProductShortName} only analyzes application code.
+
```
Comment (sbeskin-redhat) [inline docs/topics/maven-arguments.adoc:16]: @RichardHoch  
Same as above - why isn't it a flag? And a comma after "By default"

**Comment by RichardHoch** [comment]
@sbeskin-redhat  -- made the changes. Please double-check.

**Comment by sbeskin-redhat** [comment]
Looks fine

**Comment by RichardHoch** [comment]
Ready to merge.

**Pull request merged**